### PR TITLE
Add new CalibMQcTask

### PR DESCRIPTION
**Pull request by ValerieRamillien**

### CODE REVIEW

File: Modules/MUON/MID/include/MID/CalibMQcCheck.h [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  ~CalibMQcCheck() override = default;
+
+  // Override interface
+  Quality check(std::map<std::string, std::shared_ptr<MonitorObject>>* moMap) override;
+  void beautify(std::shared_ptr<MonitorObject> mo, Quality checkResult = Quality::Null) override;
+  std::string getAcceptedType() override;
+
+ private:
+  ///////////////////////////
+  float nTF = 0;
```
Comment (Barthelemy) [inline Modules/MUON/MID/include/MID/CalibMQcCheck.h:40]: The naming scheme specifies that the members should start with an `m`.  
https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html#Variable_Names

File: Modules/MUON/MID/include/MID/CalibMQcTask.h [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  void startOfActivity(Activity& activity) override;
+  void startOfCycle() override;
+  void monitorData(o2::framework::ProcessingContext& ctx) override;
+  void endOfCycle() override;
+  void endOfActivity(Activity& activity) override;
+  void reset() override;
+
+ private:
+  ///////////////////////////
+  int nTF = 0;
```
Comment (Barthelemy) [inline Modules/MUON/MID/include/MID/CalibMQcTask.h:56]: same here

File: Modules/MUON/MID/src/CalibMQcTask.cxx [hunk 3]
```diff
@@ -0,0 +1,909 @@
...
+namespace o2::quality_control_modules::mid
+{
+
+CalibMQcTask::~CalibMQcTask()
+{
+}
+
+void CalibMQcTask::initialize(o2::framework::InitContext& /*ctx*/)
+{
+  ILOG(Info, Support) << "initialize CalibMQcTask" << ENDM; // QcInfoLogger is used. FairMQ logs will go to there as well.
```
Comment (Barthelemy) [inline Modules/MUON/MID/src/CalibMQcTask.cxx:52]: Perhaps you could use `ILOG(Debug, Devel)` instead.

Comment (Barthelemy) [reply Modules/MUON/MID/src/CalibMQcTask.cxx:52]: Also for the others further down the code.

File: Modules/MUON/MID/src/CalibQcTask.cxx [hunk 4]
```diff
@@ -50,7 +50,7 @@ CalibQcTask::~CalibQcTask()
 void CalibQcTask::initialize(o2::framework::InitContext& /*ctx*/)
 {
   ILOG(Info, Support) << "initialize CalibQcTask" << ENDM; // QcInfoLogger is used. FairMQ logs will go to there as well.
-  // std::cout << "!!!! START initialize CalibQcTask !!!! " << std::endl;
+  std::cout << "!!!! START initialize CalibQcTask !!!! " << std::endl;
```
Comment (Barthelemy) [inline Modules/MUON/MID/src/CalibQcTask.cxx:53]: Please use `ILOG` instead of `cout`.

File: Modules/MUON/MID/src/CalibQcTask.cxx [hunk 5]
```diff
@@ -214,18 +214,20 @@ void CalibQcTask::initialize(o2::framework::InitContext& /*ctx*/)
...
 void CalibQcTask::startOfCycle()
 {
   ILOG(Info, Support) << "startOfCycle" << ENDM;
+  std::cout << "!!!! START startOfCycle CalibQcTask !!!! " << std::endl;
 }
 
 void CalibQcTask::monitorData(o2::framework::ProcessingContext& ctx)
 {
-  // std::cout << "!!!! START monitorData CalibQcTask !!!! " << std::endl;
+  std::cout << "!!!! START monitorData CalibQcTask !!!! " << std::endl;
```
Comment (Barthelemy) [inline Modules/MUON/MID/src/CalibQcTask.cxx:228]: logging and printing should not happen in `monitorData` (apart for the development). It results in a very large amount of logs.

**Review by Barthelemy — CHANGES_REQUESTED** [review]

**Comment by Barthelemy** [comment]
Good, thank you, just a few changes to be made (and one conflict apparently due to another PR).

**Comment by ValerieRamillien** [comment]
Hello Barthelemy,
Ok I will change it. Thank you.
Valerie.

**Review by Barthelemy — APPROVED** [review]

**Pull request merged**